### PR TITLE
test(cypress): remove no needed test cases and comments

### PR DESCRIPTION
**Pull request by KatarzynaQA**
### Proposed behaviour

Removed no needed test cases and comments
### Current behaviour

We have in the code comments `will be removed when Applitools will be implemented`. We are using Chromatic now, instead of Applitools, so

- we need to remove these comments,
- remove test cases covered by Chromatic

### Checklist

- [ ] Screenshots are included in the PR
- [ ] Carbon implementation and Design System documentation are congruent
- [ ] All themes are supported
- [ ] Commits follow our style guide
- [ ] Unit tests added or updated
- [ ] Cypress automation tests added or updated
- [ ] Storybook added or updated
- [ ] Typescript `d.ts` file added or updated

### CODE REVIEW

File: cypress/features/regression/button.feature [hunk 1]
```diff
@@ -4,16 +4,6 @@ Feature: Button component
   Background: Open Button component default page
```
Comment (DlgSHi) [inline cypress/features/regression/button.feature:4]: Please, remove also those TC:
- Scenario: Set Button icon to arrow_left_small
- Scenario Outline: Change Button icon position to <iconPosition>

File: cypress/features/regression/designSystem/drawer.feature [hunk 2]
```diff
@@ -44,16 +44,4 @@ Feature: Drawer component
   @positive
   Scenario: Set expandedWidth to 62%
     When I click on expanded-width-62-drawer Drawers arrow 1 time
```
Comment (DlgSHi) [inline cypress/features/regression/designSystem/drawer.feature:46]: I think we need also remove this TC as we have tested the expanded width for Drawer using `chromatic`

File: cypress/features/regression/button.feature [hunk 3]
```diff
@@ -33,11 +23,7 @@ Feature: Button component
       And Button background color is "<background-color>"
     Examples:
       | buttonType     | font-color         | background-color   |
-      | primary        | rgb(255, 255, 255) | rgb(0, 129, 93)    |
-      | secondary      | rgb(0, 129, 93)    | rgba(0, 0, 0, 0)   |
-      | tertiary       | rgb(0, 129, 93)    | rgba(0, 0, 0, 0)   |
       | darkBackground | rgb(0, 129, 93)    | rgb(255, 255, 255) |
```
[the remark this answers is not in the document]
Comment (KatarzynaQA) [reply cypress/features/regression/button.feature]: removed

File: cypress/features/regression/button.feature (hunk 1 above)
Comment (KatarzynaQA) [reply cypress/features/regression/button.feature:4]: removed

File: cypress/features/regression/designSystem/drawer.feature (hunk 2 above)
Comment (KatarzynaQA) [reply cypress/features/regression/designSystem/drawer.feature]: removed

**Pull request merged**